buildSrc/src/main/java/org/elasticsearch/gradle/vagrant/VagrantMachine.java

```diff
@@ -75,7 +75,7 @@ public void execute(Action<VagrantExecSpec> action) {
             execSpec.setEnvironment(System.getenv()); // pass through env
             execSpec.environment("VAGRANT_CWD", vagrantfile.getParentFile().toString());
             execSpec.environment("VAGRANT_VAGRANTFILE", vagrantfile.getName());
-            execSpec.environment("VAGRANT_LOG", "debug");
+            //execSpec.environment("VAGRANT_LOG", "debug");
```

Comment:
Can we ditch this too?

Reply:
I uncommented these for now. I will remove them in a separate, focused commit, when we are sure they are no longer useful.